assets/dev/js/frontend/utils/url-actions.js

```diff
@@ -36,6 +36,7 @@ export default class extends elementorModules.ViewModule {
 	}
 
 	runAction( url, ...restArgs ) {
+		url = decodeURI( url );
```

Comment:
I have added this, because after the security fix, we now run `encodeURI( url )` on the content templates inside the editor.

The Off Canvas Url Actions started to break inside the editor.

I have tested my fix inside the editor and on the frontend. Both are still working.

Do you think this is safe?

Reference to security fix: https://github.com/elementor/elementor/commit/dd7e04778fd27ad1398c22b7fe58a7e5053af165#diff-7370f1fa2d9e6f4e4571d527ee93660c50f3d98442c29e8176365535dabc2736R711